frontend/lib/identification/user_code_store.dart

```diff
@@ -24,7 +28,19 @@ class UserCodeStore {
 
   Future<List<DynamicUserCode>> load() async {
     const storage = FlutterSecureStorage();
-    final String? userCodesBase64 = await storage.read(key: _userCodesBase64Key);
+    String? userCodesBase64;
+    // make fake read on android - workaround for https://github.com/mogol/flutter_secure_storage/issues/653
+    if (Platform.isAndroid) {
+      String? userCodesBase64FirstRead = await storage.read(key: _userCodesBase64Key);
+      userCodesBase64 = await storage.read(key: _userCodesBase64Key);
+      // report error if first read is null but second read contains data to collect sentry data for reproduction
+      if (userCodesBase64FirstRead == null && userCodesBase64 != null) {
+        reportError('First read from secure storage is null but user data exists', null);
+      }
```

Comment:
Maybe, we can report more potential misbehavior here to get a better understanding.
Maybe something like this:
```suggestion
      if (userCodesBase64FirstRead != userCodesBase64) {
        bool firstIsNull = userCodesBase64FirstRead == null;
        bool secondIsNull = userCodesBase64 == null;
        reportError("First read from secure storage differs from second: firstNull=${firstIsNull}, secondNull=${secondIsNull}", null);
      }
```